src/main/react-front-end/src/analytics/ga.js

```diff
@@ -0,0 +1,22 @@
+import ReactGA from "react-ga";
+
+/** 
+ * Initializes the google analytics library 
+ */ 
+export function initializeReactGA() {
+  if(process.env.REACT_APP_ENV === 'production'){
+    ReactGA.initialize('UA-147970452-1');
+  }
+  else if(process.env.REACT_APP_ENV === 'development'){
+    ReactGA.initialize('UA-147976263-1');    
+  }
+}
```

Comment:
@aashish-jain we can redo this in just single line. We dont need to store environment identifiers, rather like other env vars we will store the keys and access those.